packages/components/src/components/real-wallet-card/hook/use-hover.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export const useHover = refSetter => {
```

Comment:
Already defined in `packages/components/src/hooks/use-hover.js`

Reply:
That uses `mouseover` and `mouseout` whereas the hover needs `mouseenter` and `mouseleave`, but I found that I haven't change the names so I'll do that! :)

Reply:
As commented on Nijil's card, we can pass in an extra argument to determine to which events it should listen to.